INSTALL.md

```diff
@@ -0,0 +1,157 @@
+
+choose manual download
+
+![image](https://github.com/nathmo/ESP-360-REMOTE/assets/15912256/e6ae26cc-6eb6-4dc1-970b-3d262d75b06b)
+
+the file will be compiled to a .bin that you will be able to download. you need to download the legacy format :
+
+![image](https://github.com/nathmo/ESP-360-REMOTE/assets/15912256/dbe9ecfa-dd81-4144-9331-9234900aaab9)
+
+
+![image](https://github.com/nathmo/ESP-360-REMOTE/assets/15912256/e6f7382d-27fc-405e-8511-8cb5cea83d2e)
+
+from there, give it a name and choose ESP32
+
+![image](https://github.com/nathmo/ESP-360-REMOTE/assets/15912256/8dd688a1-7d29-4f24-bf66-537c23cb7060)
+
+now return to the webpage of the ESP 360 Remote and under OTA Update browse for the file you previously downloaded and click update
+once the device as rebooted, you should see the value from the sensors and your switch
+
+![image](https://github.com/nathmo/ESP-360-REMOTE/assets/15912256/e66aeed3-7168-4010-b728-6675b3e7b089)
+
+
+
+you are done :)
+# home assistant
+TODO, please write this section if you did it
+
+# template (universal last step)
+
+here is what the template looks like. you can edit the name field and the wifi ssid + password with your
+```
+esphome:
+  name: esp360-default-1
+  friendly_name: ESP360-default-1
+
+esp32:
+  board: esp32dev
+  framework:
+    type: arduino
+
+# Enable logging
+logger:
+
+api:
+  encryption:
+    ...
+```
+you can add your own action.
+this is an example that send an IR code that turn ON and OFF a fan.
+you will need to adapt the transmit_nec function to your protocol and replace the address and command field accordingly
+```
+  - platform: template
+    name: "fan"
+    turn_on_action:
+      remote_transmitter.transmit_nec:
+        transmitter_id: IR_TX
+        address: 0xFE01
+        command: 0xE21D
+    turn_off_action:
+      remote_transmitter.transmit_nec:
+        transmitter_id: IR_TX
+        address: 0xFE01
+        command: 0xE21D
+```
+if you dont know theses, you can start by compilling the base template and go to the webpage of the ESP 360 remote, you should see a terminal that display the data from the sensors and the IR + RF transceiver
+just point the remote you want to clone toward the ESP 360  remote and look at the code display in the terminal.
+it should looks like something like this :
+
+![image](https://github.com/nathmo/ESP-360-REMOTE/assets/15912256/0b4145c7-22b1-4e91-9b0d-4f6a9d262afe)
+
+the line that we care about is the following
+```
+Received NEC: address=0xFE01, command=0xE21D
+```
+this line give you the protocol (NEC) the address and the command. if you have another protocol lookup in the ESPhome documentation the specific syntax.
+
+you can now edit the snippet with your value and add as many as you need for every function you want.
+
+(dont hesitate to read the ESPhome documentation as it will give you more detailled info and more options to configure your actions)
```

Comment:
```suggestion
> TIP: Definitely read the ESPHome documentation as it will give you more detailled information and more options to configure your actions. 
```